pkg/backup/restore/restore_manager.go

```diff
@@ -1160,7 +1160,16 @@ func (rm *restoreManager) makeSyncWarmUpJob(r *v1alpha1.Restore, tc *v1alpha1.Ti
 	}
 	resourceRequirements := getWarmUpResourceRequirements(tc)
 
+	jobAnnotations := r.Annotations
+	podAnnotations := jobAnnotations
+	jobLabels := r.Labels
```

Comment:
we have `jobLabel` definition below, so the `jobLabels` here is a little confused. I suggest to remove definition of `jobAnnotations` and `jobLabels`.

Reply:
@WangLe1321 done. Thanks